spec/services/proofing/lexis_nexis/response_spec.rb

```diff
@@ -38,7 +38,12 @@
     end
 
     context 'with a passed verification' do
-      it { expect(subject.verification_errors).to eq({}) }
+      it 'returns a hash of error' do
+        errors = subject.verification_errors
+
+        expect(errors).to be_a(Hash)
+        expect(errors).to include(:InstantVerify)
```

Comment:
maybe combine both assertions? since only hashes can have keys?

```suggestion
        expect(errors).to have_key(:InstantVerify)
```